.changeset/long-moons-worry.md

```diff
@@ -0,0 +1,7 @@
+---
+"@twilio-paste/sidebar": minor
+"@twilio-paste/website": minor
+"@twilio-paste/core": minor
+---
+
+Updating the unselected Sidebar Disclosure Header and SidebarNavigationItems.
```

Comment:
nit: add package name so changes are easier to understand in our [changelog for core
](https://paste.twilio.design/core/changelog)
```suggestion
[Sidebar] Updating the unselected Sidebar Disclosure Header and SidebarNavigationItems.
```

Reply:
I'm sorry I'm still learning on how things are done here. Thanks for helping me.

Reply:
one more changelog nit: as we don't publish the website package to npm, we don't changelog it, so that line can be removed from this file

Reply:
ok I lied! last nit here, the sidebar and core bumps can both be patches instead of minors, as we're not adding any new props or exports in this PR

Reply:
I as just about to mention the website, I missed that in my initial one. It's all good, you're doing amazing work here and we're so happy to have you 👏